configs/test/habitat_mp3d_eqa_test.yaml

```diff
@@ -1,5 +1,8 @@
 TASK:
   TYPE: EQA-v0
+  SENSORS: ['QUESTION_SENSOR']
+  POSSIBLE_ACTIONS: ['STOP', 'MOVE_FORWARD', 'TURN_LEFT', 'TURN_RIGHT', 'ANSWER']
```

Comment:
Shouldn't `ANSWER` and `STOP` be the same action? You only get the try to answer the question once.

Reply:
I was following setup we had for EQA paper, but agree STOP is redundant.

Reply:
In the setup for the paper, we had STOP == ANSWER.  The agent only every tried to answer the question when stop was called and there were no additional sensor inputs provided after STOP was called (i.e. the last frame wasn't duplicated)